jupyter_scheduler/models.py

```diff
@@ -76,11 +74,11 @@ class CreateJob(BaseModel):
     input_uri: str
     input_filename: str = None
     runtime_environment_name: str
-    runtime_environment_parameters: Optional[Dict[str, EnvironmentParameterValues]]
+    runtime_environment_parameters: Optional[Dict[str, Any]]
```

Comment:
We should not update these, these are for environment overrides, not the same as parameters in notebooks.
```suggestion
    runtime_environment_parameters: Optional[Dict[str, EnvironmentParameterValues]]
```